styleguide.config.js

```diff
@@ -87,7 +87,10 @@ module.exports = {
         {
             name: "Grid",
             content: "packages/wonder-blocks-grid/docs.md",
-            components: "packages/wonder-blocks-grid/components/*.js",
```

Comment:
Why did you get rid of this?

Reply:
I changed it b/c locally we were displaying the `Gutter` section twice (one time here, and the other time inside `Internal Components`. But now that you mention it, we are displaying that section for just for documentation purposes.

I'm going to revert this change, and get rid of the `sections` key instead.